NewTimesheet.py

```diff
@@ -0,0 +1,115 @@
+        for time_entry in timeentries_member:
+            try:
+                client_name = time_entry['tags'][0]['name']
+            except IndexError:
+                continue
+            end = time_entry['timeInterval']['end']
+            end = datetime.datetime.strptime(end, '%Y-%m-%dT%H:%M:%SZ')
+            start = time_entry['timeInterval']['start']
+            start = datetime.datetime.strptime(start, '%Y-%m-%dT%H:%M:%SZ')            
+            duration = ((end-start).total_seconds())/3600
+            dic_entry['project_name'] = time_entry['project']['name']
+            dic_entry['client_name'] = client_name
+            dic_entry['task_name'] = time_entry['task']['name']
+            dic_entry['user_name'] = users[time_entry['userId']]
+            dic_entry['description'] = time_entry['description']
+            dic_entry['duration'] = duration
+            dados_frame = dados_frame.append(dic_entry , ignore_index=True)
+            print(dados_frame)
+
+    return dados_frame
+
+
+def Organize_Data_Frame_create_excel (dados_frame):
+    '''' Organize D.Frame and create a Excel'''
+                
+    dados_frame['Horas-aulas'] = dados_frame['duration']*1.2
+    dados_frame.loc[dados_frame['project_name'] == 'Feriado',['client_name','task_name']] = 'Feriado'
+    dados_frame.loc[dados_frame['project_name'] == 'Falta Justificada',['client_name', 'task_name']] = 'Falta Justificada'
+    dados_frame_2 = pandas.DataFrame(columns = ['project_name','client_name','task_name','user_name','description','duration','Horas-aula'])
+    dados_frame_2 = dados_frame.loc[dados_frame['task_name'] == 'Outros -> comente!',['user_name','project_name','task_name','client_name','Horas-aula','description']].copy()
+    dados_frame_2 = dados_frame_2.groupby(['user_name','project_name','client_name','task_name','Horas-aula','description'],as_index=False).sum()
+    dados_frame.drop(dados_frame[dados_frame['task_name'] == 'Outros -> comente!'].index, axis=0,inplace=True)
+    del dados_frame ['duration']
+    dados_frame = dados_frame.groupby(['user_name','project_name','client_name','task_name'], as_index=False).sum()
+    dados_frame['description'] = ''
+    dados_frame.loc[dados_frame['task_name'] == 'Feriado', ['task_name']] = ''
+    dados_frame.loc[dados_frame['task_name'] == 'Falta Justificada', ['task_name']] = ''
+    timesheet_df = pandas.concat([dados_frame[['user_name','project_name','task_name','client_name','Horas-aulas','description']], dados_frame_2])
+    timesheet_df.sort_values(by = ['user_name','project_name'], inplace=True)
+    timesheet_df.to_excel('NewTimesheet.xlsx', sheet_name = 'Timesheet',index = False, float_format = "%.2f")
+
+    print("DONE!")
+
+    return timesheet_df
+
+
+def taking_timesheet_data():
+    ''' Reading Timesheet and changing for a D.Frame'''
+    Time_sheet_2019 = pandas.read_excel (r'~/Documents/NEO/clockify-to-timesheet/NewTimesheet.xlsx')
```

Comment:
This can be only:

```suggestion
    Time_sheet_2019 = pandas.read_excel ('NewTimesheet.xlsx')
```

Or it won't work in other directories.